pkgs/development/tools/azure-static-sites-client/versions.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
We should only really keep what we need but that seems to be only the date so as long as that is fixed for the version we can keep it.